indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -1,60 +1,112 @@
+                                                   data-ajax-dialog
+                                                   data-update="#email-template-manager">
+                                                   {% trans %}E-mail text{% endtrans %}
+                                                </a>
+                                            </li>
+                                        </ul>
+                                        <a href="#" class="i-button js-delete-item icon-remove icon-only"
+                                           title="{% trans %}Delete template{% endtrans %}"
+                                           data-qtip-style="danger"
+                                           data-href="{{ url_for('.email_tpl_delete', tpl) }}"
+                                           data-method="DELETE"
+                                           data-confirm="{% trans %}Are you sure you want to delete this template?{% endtrans %}"
+                                           data-title="{% trans %}Delete template{% endtrans %}"
+                                           data-update="#email-template-manager">
                                         </a>
-                                    </li>
-                                    <li>
-                                        <a data-href="{{ url_for('.email_tpl_text_edit', tpl) }}"
-                                           title="{% trans %}Edit e-mail text{% endtrans %}"
-                                           data-title="{% trans %}Edit e-mail text{% endtrans %}"
-                                           data-ajax-dialog>
-                                           {% trans %}E-mail text{% endtrans %}
-                                        </a>
-                                    </li>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="i-box-content">
+                                <div>
+                                    {% trans %}This email will be sent when the abstract is:{% endtrans %}
+                                </div>
+                                <ul>
+                                    {% for rule in tpl.rules if rule %}
+                                        <li class="notification-rule">
+                                            <strong>{{ states[rule['state'][0]] }}</strong>
+                                            {% if 'track' in rule %}
+                                                {% trans %}in{% endtrans %}
+                                                <strong>
+                                                    {% if rule['track'][0] == '*' %}
+                                                        {% trans %}any track{% endtrans %}
+                                                    {% elif rule['track'][0] is undefined %}
```

Comment:
That's correct behavior. If "anything" is allowed we don't add it to the rules dict at all (since the rule is effectively disabled since it would match anything).

`if 'track' in rule` also succeeds if `rule['track']` is an empty list (which I guess is what you test for when checking if the first element is undefined).